solutions/usaco-988.mdx

```diff
@@ -2,12 +2,15 @@
 id: usaco-988
 source: USACO Bronze 2020 January
 title: Photoshoot
-author: Ananth Kashyap
+author: Ananth Kashyap, Ben Dodge
```

Comment:
```suggestion
author: Ananth Kashyap, Ryan Chou, Ben Dodge
```
this is for me, forgot to add author name last time:
https://github.com/cpinitiative/usaco-guide/pull/2109